FAQ.md

```diff
@@ -35,7 +37,9 @@
     *   `getting-started`: issue or pull request is suitable for someone new to GitHub and/or this project
     *   `help-wanted`: a question or request for assistance
     *   `left-as-was`: marks an issue closed because the item in question will be left as it was
+    *   `merge`: issue or pull request involving upstream repository, e.g. https://github.com/swcarpentry/lesson-template
```

Comment:
`merge` is a little confusing to me. When I first see it, I think go ahead and merge the PR. Maybe `upstream` would be better?